rest/config_test.go

```diff
@@ -1656,7 +1656,7 @@ func TestLoadJavaScript(t *testing.T) {
 			}()
 			js, err := loadJavaScript(inputJavaScriptOrPath, test.insecureSkipVerify)
 			if test.errExpected != nil {
-				require.True(t, errors.As(err, &test.errExpected)) // nolint: govet // CBG-2242
+				require.True(t, assert.ErrorAs(t, err, &test.errExpected))
```

Comment:
This fixes the Go vet error but doesn't actually fix the underlying complaint that it's an incorrect usage of `errors.As` (where err is being assigned to `test.errExpected`)

Switching this to `require.ErrorIs` does seem to be causing other issues, but that's primarily because the error types being specified (e.g. `x509.UnknownAuthorityError`/`JavaScriptLoadError`) don't implement the `Unwrap` or `Is` interfaces, so cannot actually assert on the underlying error type in this way.

I would advise reworking the test such that it's doing a direct string comparison on the error message, instead of trying to use error type checking. It's simpler and works if we're able to determine the error message output in the test cases.

```go
require.EqualError(t, err, test.errExpected.Error())
```

or 

```go
require.ErrorContains(t, err, test.errExpectedString)
```